components/brave_vpn/browser/brave_vpn_service.h

```diff
@@ -29,14 +34,7 @@
 #include "url/gurl.h"
 
 #if !BUILDFLAG(IS_ANDROID)
```

Comment:
I don't see `!is_android` for brave_vpn_os_connection_api in gn so this guard looks incorrect

Reply:
done